test/e2e/pkg/infra/docker/docker.go

```diff
@@ -78,7 +78,9 @@ services:
     init: true
     ports:
     - 26656
-    - {{ if .ProxyPort }}{{ .ProxyPort }}:{{ end }}26657
+    - {{ if .RPCProxyPort }}{{ .RPCProxyPort }}:{{ end }}26657
```

Comment:
This change should also be applied below (in line 107).

Reply:
I assume the block below applies to running a different version (`UpgradeVersion`), while this block applies to the current version (`Version`). This has to be confirmed. In thesis, they should enable the same interactions, using the same ports.

However, as this is a backport, we are not supposed to introduce new behaviors or features.

Reply:
The version in `main` does not include these ports on the second block as well.

Reply:
Yes this is a bug we need to fix in main